admin/preprint_providers/forms.py

```diff
@@ -30,3 +32,27 @@ def clean_subjects_acceptable(self, *args, **kwargs):
         subjects_selected = [Subject.objects.get(id=ident) for ident in subject_ids]
```

Comment:
Thanks! Once people get more familiar with the ORM they'll stop doing that stuff.